package.json

```diff
@@ -51,6 +51,7 @@
     "@dashevo/dashcore-lib": "~0.18.12",
     "@dashevo/dashd-rpc": "~2.0.0",
     "@dashevo/dpns-contract": "~0.3.0-dev.3",
+    "@dashevo/wallet-lib": "^7.15.0-dev.2",
```

Comment:
I’m afraid to make it works we need to specify the branch with streams since it does not merge.